### DISCUSSION TITLE
How can i add?

### DISCUSSION THREAD

**Discussion by lassejlv** [cdnjs/cdnjs#14085] [General] [answered]
Hey, how can i add a libary to cdnjs? 😃

**Comment by MattIPv4** [comment, marked as answer]
👋 You can either create an issue in https://github.com/cdnjs/packages and we can create the configuration to add it, or you're also welcome to directly make a pull request there to add the config yourself (see existing config files and the contributing.md file for guidance)

**Reply by lassejlv** [reply]
Oki thanks
